automation_script/parse_xml_results.py

```diff
@@ -0,0 +1,149 @@
+import xml.etree.ElementTree as ET
```

Comment:
NIT change
Please add file banners for both files, copy from the hipify-torch repo.